tests/acceptance/blocks/verticalresults.js

```diff
@@ -66,6 +69,44 @@ class VerticalResults {
     return this._selector.exists;
   }
 
+  /**
+   * Wait for results to load on page by checking query response status and searchComplete state
+   * (timeout is set to 10 seconds)
+   */
+  async waitOnSearchComplete() {
```

Comment:
Sorry I know I just recently approved this, but after looking at the SDK half of this item maybe this waitOnSearchComplete should go into its own request logger class instead of inside the VerticalResults component's class?

Reply:
I agree, if we can move this to it's own class we could easily expand it to universal search.  Yen, if we have a new class such as `SearchRequestLogger`, we could have a `registerVerticalSearchLogger` and a `registerUniversalSearchLogger` function so that we could set it up for either type of search